### PR TITLE
arithmetic: add page

### PR DESCRIPTION
**Pull request by sahilister**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/arithmetic.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# arithmetic
+
+> Quiz on simple arithmetic.
```
Comment (navarroaxel) [inline pages/linux/arithmetic.md:3]: I think we need a word:
```suggestion
> Quiz on simple arithmetic problems.
```

File: pages/linux/arithmetic.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://manpages.debian.org/bsdgames/arithmetic.6.en.html>.
+
+- Start an arithmetic quiz:
+
+`arithmetic`
+
+- Specify one or more arithmetic operation symbols to get problems on them:
```
Comment (navarroaxel) [inline pages/linux/arithmetic.md:10]: You can highlight argument in the example description to get it a meaning, like:
```suggestion
- Specify one or more arithmetic [o]peration symbols to get problems on them:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/arithmetic.md (hunk 2 above)
Comment (sahilister) [reply pages/linux/arithmetic.md:10]: That's quirky @navarroaxel , nice!

**Review by sbrl — APPROVED** [review]
Thanks, @sahilister! That last example description is somewhat verbose, but it doesn't looks like it can particularly be helped.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**